fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/AppAndSystemHelper.kt

```diff
@@ -178,11 +178,12 @@ object AppAndSystemHelper {
     }
 
     fun isPackageInstalled(packageName: String): Boolean {
+        Log.i(TAG, "isPackageInstalled: Trying to verify that $packageName is installed")
         return try {
             val packageManager = InstrumentationRegistry.getInstrumentation().context.packageManager
             packageManager.getApplicationInfo(packageName, 0).enabled
```

Comment:
Should there be a "package is installed" log after this?

Reply:
I've added the log in the if statement [here](https://github.com/mozilla-mobile/firefox-android/pull/5613/files#diff-d0b7c98f4721fb461ed1c466d3c39d9a9e50893d37551f37091de5f926a32880R438)